### PR TITLE
[Java][Rest-assured] Changed Rest-assured params

### PR DESCRIPTION
**Pull request by viclovsky**
…een refactored

### PR checklist

- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [x] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [x] Filed the PR against the correct branch: Default: `master`.
- [x] Copied the [technical committee](https://github.com/openapitools/openapi-generator/#openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### Description of the PR
Added constants for Rest-assured params.
Made params as constants for Rest-assured.
See [issue](https://github.com/OpenAPITools/openapi-generator/issues/25).

### CODE REVIEW

File: samples/client/petstore/java/rest-assured/src/main/java/org/openapitools/client/api/FakeApi.java [hunk 1]
```diff
@@ -463,12 +463,14 @@ public TestBodyWithQueryParamsOper body(User user) {
...
 
+        public static final String QUERY_QUERY = "query";
+
         /**
          * @param query (String)  (required)
          * @return operation
          */
         public TestBodyWithQueryParamsOper queryQuery(Object... query) {
-            reqSpec.addQueryParam("query", query);
+            reqSpec.addQueryParam(QUERY, query);
```
Comment (jmini) [inline samples/client/petstore/java/rest-assured/src/main/java/org/openapitools/client/api/FakeApi.java]: Should it not be `reqSpec.addQueryParam(QUERY_QUERY, query);` ??

**Comment by jmini** [comment]
I like this a lot. Thank you!

Can it be that your update of rest-assured in `samples/` was not made with the latest template?

**Comment by viclovsky** [comment]
> Can it be that your update of rest-assured in samples/ was not made with the latest template?
Yep, just updated, thanks.

**Comment by jmini** [comment]
I tested:
```
mvn clean verify -f samples/client/petstore/java/rest-assured/pom.xml
```

And I generated a rest assured client with my spec.

Looks good to me.

**Pull request merged**